### PR TITLE
IGVF-1789-remove-pub-ids

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

**Review by khzlin — DISMISSED** [review]
looks good to me!

**Review by khzlin — DISMISSED** [review]

File: src/igvfd/schemas/in_vitro_system.json [hunk 1]
```diff
@@ -442,10 +439,6 @@
         "schema_version": {
             "default": "22"
```
Comment (khzlin) [inline src/igvfd/schemas/in_vitro_system.json:440]: upgrade to 23

File: src/igvfd/schemas/primary_cell.json [hunk 2]
```diff
@@ -66,10 +63,6 @@
         "schema_version": {
             "default": "20"
```
Comment (khzlin) [inline src/igvfd/schemas/primary_cell.json:64]: upgrade to 21

File: src/igvfd/schemas/rodent_donor.json [hunk 3]
```diff
@@ -96,10 +93,6 @@
         "schema_version": {
             "default": "13"
```
Comment (khzlin) [inline src/igvfd/schemas/rodent_donor.json:94]: upgrade to 14

File: src/igvfd/schemas/technical_sample.json [hunk 4]
```diff
@@ -60,10 +57,6 @@
         "schema_version": {
             "default": "13"
```
Comment (khzlin) [inline src/igvfd/schemas/technical_sample.json:58]: upgrade to 14

File: src/igvfd/schemas/tissue.json [hunk 5]
```diff
@@ -78,10 +75,6 @@
         "schema_version": {
             "default": "20"
```
Comment (khzlin) [inline src/igvfd/schemas/tissue.json:76]: upgrade to 21

File: src/igvfd/schemas/whole_organism.json [hunk 6]
```diff
@@ -81,10 +78,6 @@
         "schema_version": {
             "default": "23"
```
Comment (khzlin) [inline src/igvfd/schemas/whole_organism.json:79]: upgrade to 24

**Review by khzlin — CHANGES_REQUESTED** [review]
found some objects where schema version is not changed on schema level

File: src/igvfd/schemas/whole_organism.json (hunk 6 above)
Comment (ian-whaling) [reply src/igvfd/schemas/whole_organism.json:79]: fixed.

File: src/igvfd/schemas/tissue.json (hunk 5 above)
Comment (ian-whaling) [reply src/igvfd/schemas/tissue.json:76]: fixed.

File: src/igvfd/schemas/technical_sample.json (hunk 4 above)
Comment (ian-whaling) [reply src/igvfd/schemas/technical_sample.json:58]: fixed.

File: src/igvfd/schemas/rodent_donor.json (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/schemas/rodent_donor.json:94]: fixed.

File: src/igvfd/schemas/primary_cell.json (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/primary_cell.json:64]: fixed.

File: src/igvfd/schemas/in_vitro_system.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/in_vitro_system.json:440]: fixed.

File: src/igvfd/tests/test_upgrade_model_set.py [hunk 7]
```diff
@@ -5,3 +5,9 @@ def test_model_set_upgrade_1_2(upgrader, model_set_v1):
     value = upgrader.upgrade('model_set', model_set_v1, current_version='1', target_version='2')
     assert value['schema_version'] == '2'
     assert 'description' not in value
+
+
+def test_model_set_upgrade_1_2(upgrader, model_set_v3):
```
Comment (jenjou) [inline src/igvfd/tests/test_upgrade_model_set.py:10]: Test name is duplicated (probably should be 3_4)

File: src/igvfd/tests/data/inserts/curated_set.json [hunk 8]
```diff
@@ -57,9 +51,6 @@
         "documents": [
             "/documents/igvf:characterization_FACS_insert/"
         ],
-        "publication_identifiers": [
```
Comment (jenjou) [inline src/igvfd/tests/data/inserts/curated_set.json:60]: Would you be able to add "publications" into some of the inserts where "publication_identifiers" was removed? (Or other objects that didn't previously have publication_identifiers") Just want to make sure we have some examples of links to "publications".

File: src/igvfd/tests/test_upgrade_model_set.py (hunk 7 above)
Comment (ian-whaling) [reply src/igvfd/tests/test_upgrade_model_set.py:10]: fixed

File: src/igvfd/tests/test_upgrade_whole_organism.py [hunk 9]
```diff
@@ -176,3 +176,8 @@ def test_whole_organism_upgrade_22_23(upgrader, whole_organism_v22):
     assert 'product_id' not in value
     assert 'notes' in value and value['notes'].endswith(
         'Product_id 100A was removed from this sample. Lot_id 123 was removed from this sample.')
+
+
+def test_whole_organism_upgrade_23_24(upgrader, whole_organism_v23):
+    value = upgrader.upgrade('whole_organism', whole_organism_v23, current_version='23', target_version='24')
+    assert 'publication_identifiers' not in value
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/tests/test_upgrade_whole_organism.py:183]: fixed

File: src/igvfd/tests/data/inserts/curated_set.json (hunk 8 above)
Comment (ian-whaling) [reply src/igvfd/tests/data/inserts/curated_set.json:60]: I added them in the prior ticket which added the property `publications`. But I can add more here.

Comment (jenjou) [reply src/igvfd/tests/data/inserts/curated_set.json:60]: I see. In that case probably don't need to add much more, I didn't realize it was already in some inserts.

**Review by jenjou — APPROVED** [review]

**Pull request merged**